src/plone/api/content.py

```diff
@@ -473,3 +474,92 @@ def get_uuid(obj=None):
     :Example: :ref:`content_get_uuid_example`
     """
     return IUUID(obj)
+
+
+def find(context=None, depth=None, **kwargs):
+    """Find content in the portal.
+
+    Find works alike catalog(). Indexes are passing in as arguments with the
```

Comment:
The convention is to have narrative docs short and to-the-point and keep advanced usage in docstrings. So I'm +1 for this approach.